styles.css

```diff
@@ -0,0 +1,284 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+:root {
+  --primary-text: #000;
+  --secondary-text: #666;
+  --primary-bg: #fff;
+  --secondary-bg: #f7f9fa;
+  --tertiary-bg: #e0f3ff;
+  --warning-color: #ff1f13;
+  --warning-dark: #ca1910;
+  --warning-light: #ff4b42;
+  --primary-color: #39a2f2;
+  --dark-color: #1a85d6;
+  --light-color: #6cbaf7;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Merriweather', serif;
+  margin: 0;
+}
+
+/* Buttons */
+.btn {
+  font-weight: 700;
+  border-radius: 25px;
+  font-family: inherit;
+  cursor: pointer;
+}
+
+.btn-new {
+  color: white;
+  background: var(--primary-color);
+  border: none;
+  padding: 1rem 1.2rem;
+  outline: none;
+  cursor: pointer;
+}
+
+.btn-new:hover,
+.btn-new:focus {
+  outline: none;
```

Comment:
never disable outlines! - https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/#:~:text=Using%20the%20CSS%20rule%20%3Afocus,with%20the%20link%20or%20control.

You are changing the background color on focus, but still, it isn't as clear and common practice than outlining things on focus.